<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kişi Kontrol Kamera Taraması</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>391</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>40</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Kapı Kontrol</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbDurum">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>450</y>
     <width>541</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Onay Durumu: Bekleniyor</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbKamera">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>531</width>
     <height>361</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTekrarDene">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>14</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Tekrar Dene</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
